envoy/network/proxy_protocol.h

```diff
@@ -1,13 +1,24 @@
 #pragma once
 
+#include <string>
```

Comment:
I see you included this even though the header already referenced std::string (so it was just a cosmetic correction).  Should we be adding '<cstdint>' as well to pick up the std::uint8_t?

Reply:
Ah, good catch. Done. I could also remove it if you prefer.